content/posts/derisking-guide-update.md

```diff
@@ -29,5 +29,5 @@ These changes respond to current government agency needs. As part of the Office
 
 The changes also reflect the core 18F value to iterate. The guide's original authors hoped it would be revised over time. As we're able, we'll continue to do so.
 
-Check out the [new guide online](https://guides.18f.gov/derisking-government-tech/) and in [PDF](/). Share it around. Suggestions, questions, and feedback are welcome! Please reach out through the 18F Guides GitHub repo.
+Check out the [new guide online](https://guides.18f.gov/derisking-government-tech/) and in [PDF](/). Share it around. Suggestions, questions, and feedback are welcome! Please reach out through the [18F Guides GitHub repo](https://github.com/18F/guides/issues/new/choose).
```

Comment:
change: There is a hidden broken link here: the PDF link just links to the site's root path (`/`). This wouldn't be caught by our link checkers, and should be updated so this doesn't fall through the cracks.